docs/docs/install/configuration.md

```diff
@@ -0,0 +1,34 @@
+| DOMAIN_NAME | The [public URL](https://caddyserver.com/docs/caddyfile/concepts#addresses) that will be used to access Mathesar. If a https:// url is provided, the Caddy reverse proxy will attempt to [automatically setup HTTPS](https://caddyserver.com/docs/automatic-https) with "lets encrypt" for you. Set this to `localhost` if you wish Mathesar to be available on localhost only and use `http://<your_domain_name>` if you don't want caddy to automatically handle the SSL                         | Required |
+| HTTP_PORT   | The HTTP port used by caddy to listen for requests. It is recommended to use the default port `80` as features like automatic SSL [rely on it](https://caddyserver.com/docs/automatic-https#acme-challenges). If you already have a reverse proxy handling the SSL on the same system or if you are running on a non-root system, you can change it to a different port number to avoid conflicts or permission errors                                                                             | 80       |
+| HTTPS_PORT  | The HTTPS port used by caddy to listen for requests. If you want caddy to handle the SSL certificate it is highly recommended to use the default port `443` as features like automatic SSL, HTTPS redirection [rely on it](https://caddyserver.com/docs/automatic-https#acme-challenges). If you already have a reverse proxy handling the SSL on the same system or if you are running on a non-root system, you can change it to a different port number to avoid conflicts or permission errors | 443      |
+
+### Database Configuration
+| Name              | Description                                                                                                                                                                                                                                                     | Defaults          |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| POSTGRES_PORT     | The port the postgres database listens on                                                                                                                                                                                                                       | `5432`            |
+| POSTGRES_DB       | The name of the default database to create                                                                                                                                                                                                                      | `mathesar_django` |
```

Comment:
> The name of the default database to create

1. If there's an existing database, we don't create it. We should mention that we connect to this db, and if it doesn't exist, we create it.
2. We should explicitly state that this DB is different from the user databases, and that we use this for internal purposes.

Reply:
> We should explicitly state that this DB is different from the user databases, and that we use this for internal purposes.

The variable is meant to just create a new database, it is up to the user to use it however he wants. The environment variable responsible for configuring the database to be used for the internal purposes is the `DJANGO_DATABASE_URL`